lib/l10n/app_en.arb

```
@@ -2772,5 +2776,17 @@
   "dragToReorder": "Drag to Reorder",
   "@dragToReorder": {
     "description": "Label shown on drag handles of track list  tiles in the queue list"
+  },
+  "addDownloadLocationsErrorPrompt": "Writing to selected directory failed.  You should modify permissions or choose another location.",
+  "@addDownloadLocationsErrorPrompt": {
+    "description": "Label shown on popup dialog when adding a download location when writing songs fails"
+  },
+  "androidImageErrorPrompt": "WARNING: We are unable to write images to the android Music folder due to OS restrictions.  You should choose another location.",
```

If our plan is to keep this "supported" (as in possible), can we eventually ignore any image-related errors for such locations?  
Alternatively we could offer a choice to either not download images (since they would fail either way), or download them into a separate location (either external or internal), although that's a more involved solution I guess.  
Not blocking, just want to discuss this :)

---

I don't really want to add code supporting this sort of usage.  I was hesitant to block something that was theoretically doable before, but maybe we should just prevent this completely if its basically guaranteed to spam errors?

---

I agree, and this is the kind of change I'd like to make before releasing to stable. It will probably still lead to errors when users migrate from current stable to the redesign. Maybe we could prompt them to migrate to another directory, moving the existing files instead of requiring a redownload?

For newly added download locations we could suggest people to simply create a "Finamp" or "Music" directory within the "Downloads" directory, that works fine.

---

@Komodo5197 so we'll just throw an error if the write check fails, instead of showing a warning?